server/issue.go

```diff
@@ -339,7 +339,7 @@ func (s *Server) composeIssueRelationship(ctx context.Context, issue *api.Issue)
 }
 
 func (s *Server) createIssue(ctx context.Context, issueCreate *api.IssueCreate, creatorID int) (*api.Issue, error) {
-	if issueCreate.Type == api.IssueDatabaseCreate {
+	if len(issueCreate.Pipeline.StageList) <= 0 {
```

Comment:
Would be better to leave a comment here. I guess it means if frontend does not pass the stageList, we will generate it from backend?

Also len () <= 0 seems a bit odd